src/index.js

```diff
@@ -7,43 +7,61 @@ import transformCall from './transformers/call';
 import transformImport from './transformers/import';
 
 
-const defaultBabelExtensions = ['.js', '.jsx', '.es', '.es6'];
-export const defaultExtensions = defaultBabelExtensions;
```

Comment:
Oh actually I think you're right. Because the plugin will work on every file now, I believe it also make it simpler for the eslint plugin.

👍